api/src/main/java/org/openmrs/scheduler/Schedule.java

```diff
@@ -200,17 +198,6 @@ public void setStartTime(Date startTime) {
 	 * 
 	 * @param startTime start time for the task
 	 */
```

Comment:
@Nuwantha remove this javadoc which was the belonging to this commented out code.